classes/controller/FrontController.php

```diff
@@ -1454,11 +1454,15 @@ public function setTemplate($default_template)
      * specific controller.
      *
     * @since 1.5.0.13
+    * @param string $defaul_template
     * @return string|bool
     */
-    public function getOverrideTemplate()
+    public function getOverrideTemplate($default_template)
```

Comment:
Hello @aleeks. Actually, I ran a global search for "getOverrideTemplate()" before submitting this PR but forget to check what was an empty result. Unless keeping function signature is for another purpose?